src/store/migrations/02-set-permissions.js

```diff
@@ -0,0 +1,7 @@
+export default state => {
+  const newState = { ...state };
+
+  newState.permissions = {};
```

Comment:
The problem is when user has previous permissions (he logined in 0.3.3) which had `address`, 'messageSign', 'transactionSignLimit` they'll have this in new hosts list, because it just takes all state of `permissions` module. I've decided to specially reset their permissions state to avoid that.